packages/meditrak-server/src/routes/accessRequests/GETAccessRequests.js

```diff
@@ -44,15 +46,22 @@ export class GETAccessRequests extends GETHandler {
   }
 
   async findRecords(criteria, options) {
+    const dbConditions = criteria;
+    if (!hasBESAdminAccess(this.accessPolicy)) {
+      // If we don't have BES Admin access, add a filter to the SQL query
+      const countryList = this.accessPolicy.getEntitiesByPermission(
```

Comment:
```suggestion
      const accessibleCountryCodes = this.accessPolicy.getEntitiesByPermission(
```

Reply:
Renamed. I've also moved the creation of SQL filters into the permissions files to make this read a bit cleaner.